## Doppler Settings for VLA $H_2O$ Observations
### Kimber, C; Lovato, A

In [3]:
pylab inline

Populating the interactive namespace from numpy and matplotlib


In [4]:
nu0_h2o       = 22235.0798E-3     # Ghz
sun_lsr       = 20E3              # m/s
lkha234_lsr   = 14E3              # m/s
m_per_au      = 1.496E11          # m/au
orbital_circ  = m_per_au * 2 * pi # m
day           = 24 * 3600         # sec
year          = 365 * day         # sec
orbital_vel   = orbital_circ/year # m/s
earth_circ    = 2 * pi * 6400E3   # m
rot_vel       = earth_circ/day    # m/s
c             = 3E8               # m/s

In [5]:
print("The maximum velocity of the rotation of the Earth is %2.2E m/s" % rot_vel)
print("And the maximum velocity of Earth's orbit is %2.2E m/s" % orbital_vel)

The maximum velocity of the rotation of the Earth is 4.65E+02 m/s
And the maximum velocity of Earth's orbit is 2.98E+04 m/s


In [7]:
v_max         = sun_lsr + lkha234_lsr + orbital_vel + rot_vel
delta_nu_max  = nu0_h2o * (-v_max/c)
print("The maximum doppler shift is %2.2E GHz." % delta_nu_max)
print("The minumum bandwidth we should observe with is between %2.4f GHz and %3.4f GHz" % (nu0_h2o + delta_nu_max, nu0_h2o - delta_nu_max))

The maximum doppler shift is -4.76E-03 GHz.
The minumum bandwidth we should observe with is between 22.2303 GHz and 22.2398 GHz


Given that $$\frac{\Delta \nu}{\nu_o} = \frac{-v}{c}$$ 

And given that the rest frequency of $H_2O$, according to NASA, is 22.3 GHz, we were tasked with finding the minimum bandwidth for the VLA to observe the young star LkH$\alpha$. We know the motion of the star with respect to the local standard of rest is 14 km/s and we know the motion of the sun with respect to the local standard of rest is 20 km/s. First we find the maximum orbital and rotational velocities of the Earth, which are calculated in the section labeled 'Constants'. Next, we find the absolute maximum velocity the Earth could have away from the star, and calculate how that would affect the observed frequency. Finally, we just add and subtract that maximum frequency shift from the rest frequency of water to find our observing bandwidth.

We found $\Delta \nu_{max}$ was $4.76 \times 10^{-3}$GHz, so we should observe between $22.2303$ and $22.2398$ GHz.